docs/user-documentation/content_types.md

```diff
@@ -0,0 +1,126 @@
+# Create / Update a Content Type
+
+!!! tip The screenshots and pre-existing data in this documentation assume that you are using the [Islandora Demo](https://github.com/Islandora-CLAW/islandora_demo) configuration. 
+
+!!! tip This page will address how to create and modify ingest forms (or rather, content types) via the GUI. For help working with forms via the API, please check out the [Further Reading](#further-reading) section for links to more advanced Drupal documentation.
+
+Since objects in Islandora 8 are stored in Drupal as Nodes, we use the standard Drupal Content Types system to create and edit our ‘ingest forms’ [Content Types in Drupal 8](https://www.drupal.org/docs/8/administering-drupal-8-site/managing-content-0/working-with-content-types-and-fields). Islandora 8 forms are Drupal forms, and if you are already familiar with Drupal Field UI, you’re already well equipped to create and modify your own ingest forms in Islandora 8.
+
+This page will address how to create and modify ingest forms (or rather, content types) via the GUI. For help working with forms via the API, please check out the Further Reading section for links to more advanced Drupal documentation.
+
+## Modify a Content Type
+
+If you have deployed your Islandora 8 with the Islandora Demo configuration, you will already have a Repository Item content type available, with pre-configured fields. In the Admin menu, go to Structure >> Content Types and find the Repository Item content type. Select *Manage Fields*.
+
+![a screenshot of the Add Content Type page](/docs/assets/islandora8_managefields.png)
+
+You will see a list of the fields that are already available in this content type. 
+
+### Add a Field
+
+Let’s add a new field where a user can indicate if the repository item needs to be reviewed:
+
+1. Click “Add Field”
+1. Since this is a “yes/no” decision, we will use “Boolean” and give the field a name. [List of Drupal 8 FieldTypes, FieldWidgets, and FieldFormatters](https://www.drupal.org/docs/8/api/entity-api/fieldtypes-fieldwidgets-and-fieldformatters)
```

Comment:
Change:` we will use “Boolean” and give the field a name.`
To: choose "Boolean" from the dropdown menu and give the Label field a name.

Reply:
@manez, we still need the "Choose Save and continue" text @librarychik mentioned.